pkg/controller/controlplane/valuesprovider.go

```diff
@@ -667,14 +686,3 @@ func getLimitValidationWebhookControlPlaneChartValues(cluster *extensionscontrol
 
 	return values, nil
 }
-
-// Data for configuration of IDM-API WebHook (deployment to be done!)
-type UserDirectory struct {
```

Comment:
Unrelated but can be removed as discussed with @droid42

Reply:
can you please create a issue of this topic to be sure not to miss this.